specification/cognitiveservices/data-plane/WebSearch/v1.0/WebSearch.json

```diff
@@ -0,0 +1,1211 @@
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "q",
+            "x-ms-client-name": "Query",
+            "in": "query",
+            "description": "The user's search query term. The term may not be empty. The term may contain Bing Advanced Operators. For example, to limit results to a specific domain, use the site: operator.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "responseFilter",
+            "x-ms-client-name": "ResponseFilter",
+            "in": "query",
+            "description": "A comma-delimited list of answers to include in the response. If you do not specify this parameter, the response includes all search answers for which there's relevant data. Possible filter values are Computation, Images, News, RelatedSearches, SpellSuggestions, TimeZone, Videos, Webpages. Although you may use this filter to get a single answer, you should instead use the answer-specific endpoint in order to get richer results. For example, to receive only images, send the request to one of the Image Search API endpoints. The RelatedSearches and SpellSuggestions answers do not support a separate endpoint like the Image Search API does (only the Web Search API returns them). To include answers that would otherwise be excluded because of ranking, see the promote query parameter.",
+            "required": false,
+            "type": "array",
+            "items": {
+              "type": "string",
+              "enum": [
+                "WebPages",
+                "Images",
+                "SpellSuggestions",
+                "News",
+                "RelatedSearches",
+                "Videos",
+                "WebPageGroupings",
+                "Computation",
+                "TimeZone"
+              ],
+              "x-ms-enum": {
+                "name": "AnswerType",
+                "modelAsString": false
```

Comment:
I'd go with `modelAsString: true` here to avoid breaking changes later.

Reply:
Done